lib/ecto/query/builder.ex

```diff
@@ -282,6 +282,32 @@ defmodule Ecto.Query.Builder do
     {{:{}, [], [:coalesce, [], [left, right]]}, params_acc}
   end
 
+  def escape({:over, _, [aggregate]}, type, params_acc, vars, env) do
+    {aggregate, params_acc} = escape(aggregate, type, params_acc, vars, env)
+    {{:{}, [], [:over, [], [aggregate, nil]]}, params_acc}
+  end
+
+  def escape({:over, _, [aggregate, {window_name, _, nil}]}, type, params_acc, vars, env) when is_atom(window_name) do
```

Comment:
I am wondering... now that we changed to `windows`, should we make the window name an atom here instead of a binding? So it would be `over(count(p.foo), :window_name)`?

Reply:
I see that you support the atom clause below, so maybe it is a matter of getting rid of this one.